torch/distributed/tensor/experimental/_attention.py

```diff
@@ -33,6 +33,19 @@
 
 __all__ = ["context_parallel"]
 
+
+class _CausalBehavior(Enum):
+    SKIP = None
+    NOT_IS_CAUSAL = False
+    IS_CAUSAL = True
+
+
+class _RotateMethod(Enum):
+    ALL_TO_ALL = auto()
+    ALL_GATHER = auto()
+    SEND_RECV = auto()
```

Comment:
double confirm: this is not implemented yet, right? If user specifies this rotate method, will it error out?

Reply:
ye, we should remove it. SEND_RECV is the same as ALL_TO_ALL.